website/docs/docs/cloud-integrations/semantic-layer/power-bi.md

```diff
@@ -11,7 +11,7 @@ The Power BI integration enables you to query the <Constant name="semantic_layer
 
 :::tip
 The <Constant name="semantic_layer" /> Power BI integration is currently in private beta. To join the beta, contact your account representative.
```

Comment:
is PBI for SL preview now or still beta?

Reply:
according to [release notes](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes#2025-dbt-launch-showcase), PBI is in preview, not beta.

Reply:
if so, do the installer steps need updating?